.gitignore

```diff
@@ -26,7 +26,6 @@
 .idea/**/gradle.xml
 .idea/**/libraries
 .gradle/
-gradle.properties
```

Comment:
Why did you remove gradle.properties from .gitignore file?

Reply:
To make AndroidX Test library works, we need to add some statements about it in `gradle.properties` and it must be included in the project.
You can check this out [here](https://developer.android.com/jetpack/androidx#using_androidx_libraries_in_your_project) :)

Reply:
@sirano11 

I added again `gradle.properties` at this [commit](https://github.com/klaytn/caver-java/pull/240/commits/466248cff26478e2ebb20ad0c5c2c7fe7ae1b62f) 😃